setup.env

```diff
@@ -1,38 +1,70 @@
-### Change the LSST_SDK_INSTALL and OSPL_HOME to the actual locations and uncomment
+### Script to setup SAL environment. Default config assumes ts_opensplice is
+### available in the same directory as ts_sal.
 ###
-### export LSST_SDK_INSTALL=/home/saluser
-### export OSPL_HOME=$LSST_SDK_INSTALL/OpenSpliceDDS/V6.9/HDE/x86_64.linux
+### To get required projects from git, do:
 ###
-### export PYTHON_BUILD_VERSION=3.6m
-### export PYTHON_BUILD_LOCATION=/usr/local
+### mkdir lsst && cd lsst  # or some other directory
+### git clone https://github.com/lsst-ts/ts_opensplice
+### git clone https://github.com/lsst-ts/ts_sal
+### source ts_sal/setup.env
+###
+### You should then be able to run salgenerator and get its help message.
+###
+### If needed, please change the LSST_SDK_INSTALL and OSPL_HOME to the actual locations before
```

Comment:
Please consider inserting "environment variables" somewhere in the above text.

Also please modify the text to explain if you want the user to edit their definition in this file, or define the environment variable outside of this script before sourcing this file (please not "running the script").

Reply:
New header:

# Script to setup SAL environment. Default config assumes ts_opensplice is
# available in the same directory as ts_sal. You don't need to edit this file
# for the default config described in README.md. Please consult README.md for
# details.
#
# If needed, please change the LSST_SDK_INSTALL and OSPL_HOME to the actual
# locations before running the script.